schema/SchA.json

```diff
@@ -436,7 +436,7 @@
                 "FIELD_FORM_ASSOCIATION": "F3 | F3X | F3P"
             }
         },
-        "contribution_amount_{f3l_bundled}": {
+        "contribution_amount": {
```

Comment:
The property name in the fec_recommended array needs to be removed as well:

    "fec_recommended": [
        "contributor_street_1",
        "contributor_city",
        "contributor_state",
        "contributor_zip",
        "contribution_date",
        "contribution_amount_{f3l_bundled}"
    ],